resources/web/signaldata-upload.lib.xml

```diff
@@ -0,0 +1,17 @@
+<libraries xmlns="http://labkey.org/clientLibrary/xml/">
+<library compileInProductionMode="false">
```

Comment:
compileInProductionMode was inherited from the HPLC module version https://github.com/LabKey/HPLC/blob/develop/resources/web/hplc-upload.lib.xml